Refactor using field serializer

ondenman opened the pull request:
Instead of returning a list of Member objects from `#to_a`, Members now returns the members using field_serializer.

ondenman committed:
Remove LowerHouseMembers

Consolidating LowerHouseMembers with Members. (I decided that
the specialization was unnecessary as we were only scraping
the single house at this moment.)

A step towards refactoring Members to use FieldSerializer.

ondenman committed:
Remove Members#format

Moved logic into to_a

Step towards refactoring Members to use field_serializer.

ondenman committed:
Rename MemberFormatter to Member

Member is a better representation of what this class represents:
a member of the legislature.

ondenman committed:
Use FieldSerializer in Members

Members now returns the list of Member objects
in :members field.

tmtmtmtm commented on lib/members.rb:
```diff
@@ -1,17 +1,22 @@
-    format(all_members)
+  field :members do
+    lower_house_members.map do |mem|
+      Member.new(mem).to_h
```
It seems a little odd to coerce these to Hashes here, rather than just returning the underlying objects.

tmtmtmtm commented on lib/members.rb:
```diff
@@ -1,17 +1,22 @@
 
-  def to_a
-    format(all_members)
+  field :members do
```
There's a bit of a mismatch here with what this class is representing. If this only providing information on lower house members, and all are available at the source, then _either_ the class should make that apparent, _or_ the method. (i.e. when we later want to also get the upper house, we should either be adding a new class or a new method, not modifying either of these).

ondenman committed:
Store Member objects not hashes

[:members] now stores Member objects rather than hashes.

ondenman committed:
Add :members_of_the_lower house field

:members_of_the_lower house replaces :members so that it is
apparent what this list represents.
